converter/convert-to-ttl.rb

```diff
@@ -369,12 +370,12 @@ def parse_kandidaten(lijsten)
           log.info "invalid date #{row["geboortedatum"]} for rrn: #{row["RR"]}, row: #{index} "
           geboortedatum = nil
         end
-        unless row['Rrvoornaam']
-          log.info "missing Rrvoornaam for rrn: #{row["RR"]}, row: #{index}"
+        unless row['RRvoornaam']
```

Comment:
the example file i received had this brought in line with the RRAchternaam already so i changed it here too